packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -579,11 +600,13 @@ export const useTransactionHistory = (
       const startIndex = _page * MAX_BATCH_SIZE
       const endIndex = startIndex + MAX_BATCH_SIZE
 
-      return Promise.all(
+      const txs = await Promise.all(
         dedupedTransactions
           .slice(startIndex, endIndex)
           .map(transformTransaction)
       )
+
+      return txs.filter(tx => tx !== null) as MergedTransaction[]
```

Comment:
Do we need this? We run `validateSourceAndDestinationChainIds` in `useOftTransactionHistory` so we shouldn't end up with null.

 And if it's still the case, we could filter those out inside the oft hook?